### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by mitogh**

### CODE REVIEW

**Comment by mitogh** [comment]
@zzuga @adamf321 Let me know if you have any comments on this.

Thank you.

**Comment by zzuga** [comment]
Whats up with wp-acf?

```
    - moxie-leean/wp-widgets dev-master requires moxie-leean/wp-acf dev-master -> no matching package found.
```

Are we waiting for an update or is wp-acf really missing here?

Apart from that, looks fine. Much cleaner.

**Comment by adamf321** [comment]
There's something going on with composer - you need to override the minimum version level in the root project, for some reason it doesn't accept the dependency in the sub project.
@mitogh do you know why?

File: src/StaticApi/Menus.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public static function get_sub_menu_items( $menu_items, $parent_id ) {
 		$items = [];
 
 		foreach ( $menu_items as $menu_item ) {
-			if ( $parent_id == $menu_item->menu_item_parent ) {
+			if ( $parent_id === $menu_item->menu_item_parent ) {
```
Comment (adamf321) [inline src/StaticApi/Menus.php]: Did you test this @mitogh ?  If I remember rightly I had to use == because one is a string and the other an int.

**Comment by mitogh** [comment]
mmm I’m going to take a look at this issues.

Thanks.

**Comment by mitogh** [comment]
@adamf321 @zzuga 

Comments has been addressed, let me know if you have any other comment before moving forward.

Thanks.

**Pull request merged**